lib/rubocop/cop/lint/number_conversion.rb

```diff
@@ -120,6 +122,11 @@ def correct_sym_method(to_method)
           "{ |i| #{body} }"
         end
 
+        def remove_parentheses(corrector, node)
+          corrector.replace(node.loc.begin, ' ')
+          corrector.remove(node.loc.end)
+        end
```

Comment:
We have two other `remove_parentheses` methods defined on various cops (although they work slightly differently in each case) maybe it'd be good to normalize that at some point.

Reply:
Yeah. I didn't normalize here because each one has different behavior, but I'll look at it separately :-)